docs/installation/server-config/security.rst

```diff
@@ -1,4 +1,225 @@
+        password="<password>";
+
+-------------------------------------------
+Configuring Authentication of KSQL to Kafka
+-------------------------------------------
+
+This configuration connects to an Apache Kafka cluster using SASL and SSL and integrate with |c3-short|
+using the :ref:`Control Center interceptors <controlcenter_clients>`.
+
+
+.. code:: bash
+
+    # Confluent Monitoring Interceptors for Control Center streams monitoring
+    producer.interceptor.classes=io.confluent.monitoring.clients.interceptor.MonitoringProducerInterceptor
+    consumer.interceptor.classes=io.confluent.monitoring.clients.interceptor.MonitoringConsumerInterceptor
+
+    # Confluent Monitoring interceptors SASL / SSL config
+    confluent.monitoring.interceptor.security.protocol=SASL_SSL
+    confluent.monitoring.interceptor.ssl.truststore.location=/etc/kafka/secrets/kafka.client.truststore.jks
+    confluent.monitoring.interceptor.ssl.truststore.password=confluent
+    confluent.monitoring.interceptor.ssl.keystore.location=/etc/kafka/secrets/kafka.client.keystore.jks
+    confluent.monitoring.interceptor.ssl.keystore.password=confluent
+    confluent.monitoring.interceptor.ssl.key.password=confluent
+    confluent.monitoring.interceptor.sasl.jaas.config=org.apache.kafka.common.security.plain.PlainLoginModule required username="client" password="client-secret";
+    confluent.monitoring.interceptor.sasl.mechanism=PLAIN
+
+The exact settings you need depend on what SASL mechanism your Kafka cluster is using and how your SSL certificates
+are signed. For more information, see the `Security section of the Kafka documentation <http://kafka.apache.org/documentation.html#security>`__.
+
+.. _config-security-ksql-acl:
+
+-------------------------------------------------
+Configuring Authorization of KSQL with Kafka ACLs
+-------------------------------------------------
+
+You can use KSQL with Apache Kafka clusters that are secured with ACLs. The behavior depends on whether the KSQL cluster is
+interactive or non-interactive.
+
+The :ref:`Kafka Authorizer <kafka_authorization>` (``SimpleAclAuthorizer``) terminology is used to describe the ACLs.
+Each ACL is made up of these parts:
+
+Resource
+    A resource is an object that adheres to the permissions defined by the ACL. A resource is comprised of a resource type
+    and resource name:
+
+    - ``RESOURCE_TYPE``, for example ``TOPIC`` or consumer ``GROUP``.
+    - Resource name, where the name is either specific or the wildcard ``*``, meaning all resources of this type.
+
+    The ``CLUSTER`` resource type does not require a resource name because it refers to the entire Kafka cluster.
+
+Operation
+    The operation that is performed on the resource, for example ``READ``.
+
+Permission
+    The type of access an ACL allows (``ALLOW``) or denies (``DENY``) access.
+
+Principal
+    An authenticated user or group. For example, ``"user: Fred"`` or ``"group: fraud"``. The identity of Kafka clients
```

Comment:
I think we can drop this sentence:
> The identity of Kafka clients is the user principal which represents a specific authenticated user in a secure cluster.

In the paragraph below we go on to say that all ACLs discussed are for the user KSQL has authenticated as, (or a group containing the user).